src/drivers/net/intel.h

```diff
@@ -319,6 +327,8 @@ enum intel_flags {
 	INTEL_NO_ASDE = 0x0008,
 	/** Reset may cause a complete device hang */
 	INTEL_RST_HANG = 0x0010,
+	/** Reset doesn't affect Packet Buffer Size registers */
+	INTEL_NO_PBSIZE_RST = 0x0020,
```

Comment:
We have the existing flag `INTEL_NO_PHY_RST` which effectively means "please do not perform a PHY reset in software".  This new flag effectively means "please do perform a PBSIZE reset in software".  The naming is therefore potentially confusing.

The most similar existing flag seems to be `INTEL_PBS_ERRATA` (which looks like a very similar bug for some of the ICH chips).

I would suggest changing this name to minimise potential confusion.

Reply:
I've changed it to `INTEL_SET_PBSIZE` to hopefully be more literal, although I am still open to suggestions there.

Reply:
Just circling back to this, would you prefer `INTEL_PBSIZE_RST`, as in "please reset the PBSIZE registers"? I initially didn't want to add `ERRATA` since the hardware bug is not listed as a errata, sighting or bug. The register chapter just says "these only reset on power up" at the beginning.

Reply:
Good point on it not being technically an errata.  I think `INTEL_PBSIZE_RST` with the comment "PBSIZE registers must be explicitly reset" or similar is a good call.

Reply:
I've swapped to `INTEL_PBSIZE_RST` and added your comment suggestion in the most recent revision